src/index.js

```diff
@@ -17,6 +17,9 @@ export default class BlobbyS3 {
       style: 'path',
       agent: undefined // use http.globalAgent by default
     }, opts);
+    if (this.options.port === 'none') {
```

Comment:
what is the value of this?

Reply:
In our CEPH config file, we don't have `port` filed, using default port (80) will cause error while getting photos

Reply:
What is the port then? It's required one way or another, be it defaults or otherwise.

Reply:
hmm, we don't have `port` value in our CEPH config, will ask CEPH team if they know it

Reply:
get the port from ceph team, will update blobby-api config files and close this PR